openshift/hac-dev
# Remove test prefix in common resource labels

## Fixes 


https://issues.redhat.com/browse/HAC-2628

## Description


Remove test prefix in common resource labels like application/component etc and update PAC label/annotation prefix. Test team has removed these prefix and also modified PAC labels as part of https://issues.redhat.com/browse/HACBS-1331

From:
```
        'test.appstudio.openshift.io/application': 'test-application',
```
To:
```
        'appstudio.openshift.io/application': 'test-application',
```
## PAC label change

from
```
pipelinesascode.tekton.dev/sha
```
to
```
pac.test.appstudio.openshift.io/sha
```

## Type of change





## Screen shots / Gifs for design review 


**Before:**
![Screenshot 2022-12-15 at 6 23 09 PM](https://user-images.githubusercontent.com/9964343/208382813-720fd665-72c1-4e3c-b898-6c2b5e14025d.png)

**After:**
![Screenshot 2022-12-15 at 6 21 14 PM](https://user-images.githubusercontent.com/9964343/208382800-6de73094-fde4-4cec-94f0-dec7ad105eec.png)

## How to test or reproduce?







1. Add integration test to your namespace and check the status of the test in lifecycle visualization.

## Browser conformance:
